modules/exploits/multi/http/tomcat_jsp_upload_bypass.rb

```diff
@@ -0,0 +1,116 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
```

Comment:
Please address `msftidy` warnings and add documentation.

https://github.com/rapid7/metasploit-framework/wiki/Writing-Module-Documentation

```
[*] Running msftidy.rb in ./.git/hooks/post-merge mode
--- Checking new and changed module syntax with tools/dev/msftidy.rb ---
modules/exploits/multi/http/tomcat_jsp_upload_bypass.rb - [WARNING] Explicitly requiring/loading msf/core is not necessary
modules/exploits/multi/http/tomcat_jsp_upload_bypass.rb - [WARNING] Explicitly using self.class in register_* is not necessary
modules/exploits/multi/http/tomcat_jsp_upload_bypass.rb:70 - [WARNING] Spaces at EOL
------------------------------------------------------------------------
------------------------------------------------------------------------
[*] This merge contains modules failing msftidy.rb
[*] Please fix this if you intend to publish these
[*] modules to a popular metasploit-framework repo
------------------------------------------------------------------------
```